doc/openapi/v3/device-onvif-camera.yaml

```diff
@@ -8216,6 +8311,19 @@ components:
       - ConfigurationToken
       - RuleName
       type: object
+    analytics_GetAnalyticsModuleOptions:
+      properties:
+        ConfigurationToken:
+          description: Reference to an existing AnalyticsConfiguration.
```

Comment:
Is this the right description? I see for `ConfiguationToken` couple of places down where it describes as some `Reference` but in other places it starts of describing as a `token` such as https://github.com/edgexfoundry/device-onvif-camera/pull/361/files#diff-9d76c2f3691e66088d86858515a5e47b0c77fe7b6f7bc178f35430a477ead31cR12287

Reply:
@presatish I would say that yes it is the right description. It is copied directly from the onvif xml schemas (just converted to json-schema/yaml). In my eyes, a `Token` is also a `Reference` and an `ID`. I believe they use these terms interchangeably as well. 

Also, I believe the token you linked is not even the same token, since one is for metadata and the other is analytics, but I could be wrong here.